src/ansiblelint/rules/meta_runtime.py

```diff
@@ -32,8 +32,9 @@ class CheckRequiresAnsibleVersion(AnsibleLintRule):
     # Refer to https://access.redhat.com/support/policy/updates/ansible-automation-platform
     # Also add devel to this list
     supported_ansible = ["2.14.", "2.15.", "2.16."]
+    supported_ansible_examples = [f">={x}0" for x in supported_ansible]
     _ids = {
-        "meta-runtime[unsupported-version]": "'requires_ansible' key must be set to a supported version - 2.13.x, 2.14.x, 2.15.x",
+        "meta-runtime[unsupported-version]": f"'requires_ansible' key must refer to a currently supported version such: {', '.join(supported_ansible_examples)}",
```

Comment:
```suggestion
        "meta-runtime[unsupported-version]": f"'requires_ansible' key must refer to a currently supported version such as: {', '.join(supported_ansible_examples)}",
```

Reply:
I'm still not seeing the missing 'as' here. There might be something else you're going for but the text doesn't parse right to me without it